server/scripts/standalone/service-standalone.sh

```diff
@@ -15,25 +23,16 @@ export PLATFORMSDK_DIR=${EXTENDER_DIR}/platformsdk
 # We need access to the toolchain binary path from within the application
 export PATH=${PLATFORMSDK_DIR}/XcodeDefault12.5.xctoolchain/usr/bin:/usr/local/bin:${PATH}
 
-# Get the Spring profile from the environment file
-if [[ -f "${ENVIRONMENT_FILE}" ]]
-then
-    while IFS='=' read -r key value
-    do
-        if [[ "${key}" = "profile" ]]; then
-            PROFILE=$(echo ${value})
-            echo "Using profile $PROFILE"
-        fi
-    done < "${ENVIRONMENT_FILE}"
-else
-    echo "Error! Environment file ${ENVIRONMENT_FILE} not found, exiting."
-    exit 1;
-fi
+export MANIFEST_MERGE_TOOL=${EXTENDER_DIR}/current/manifestmergetool.jar
 
-if [[ -z ${PROFILE} ]]; then
-    echo "Error! No Spring profile set in environment file ${ENVIRONMENT_FILE}, exiting."
-    exit 1;
-fi
+# From Dockerfile
+export XCODE_12_VERSION=12.5
+export MACOS_10_15_VERSION=10.15
+export IOS_14_VERSION=14.5
+export LIB_TAPI_1_6_PATH=/usr/local/tapi1.6/lib
+export MACOS_11_VERSION=11.3
+export XCODE_12_CLANG_VERSION=12.0.5
+export SWIFT_5_VERSION=5.0
```

Comment:
I'd like a comment in the Dockerfile referencing this file as well, since it's duplicated code and easy for it to diverge

Reply:
Good point. I've updated the PR with this addition to the Dockerfile